adapter_test.go

```diff
@@ -339,6 +350,22 @@ func TestAdapterWithCustomTable(t *testing.T) {
 	testFilteredPolicy(t, a)
 }
 
+func TestAdapterWithMulDb(t *testing.T) {
+	dsn := "root:@tcp(127.0.0.1:3306)/casbin"
+	dbPool, err := initDbResolver([]gorm.Dialector{mysql.Open(dsn)}, []string{"casbin"})
+
+	if err != nil {
+		panic(err)
+	}
+
+	a := initAdapterWithGormInstanceByMulDb(t, dbPool, "casbin", "", "casbin_rule")
+	testAutoSave(t, a)
+	testSaveLoad(t, a)
+
+	a = initAdapterWithGormInstanceByMulDb(t, dbPool, "casbin", "", "casbin_rule")
+	testFilteredPolicy(t, a)
```

Comment:
Multiple DB?

Reply:
Because i don't know how to create multiple DB by Github Action CI,So only one DB is used.

Reply:
In fact, gorm adapter could create db automatically. You could take advantage of this. 
https://github.com/casbin/gorm-adapter/blob/b448b9d5871c5c7f2b34cf7868da5e52d72e4e5a/adapter.go#L92

Reply:
I'll try

Reply:
It is feasible.
I just use ```NewAdapter``` to create new db
https://github.com/casbin/gorm-adapter/blob/fbbb3193045a3f1bb302b3fb37fcd107a4e10db7/adapter_test.go#L354-L359

Reply:
@tangyang9464 I saw your test code, and I thought it's not good enough
![image](https://user-images.githubusercontent.com/52988625/122077927-c393e880-ce2e-11eb-9fa1-34ebf9d9a958.png)
It should show the relationship between two adapters.

Reply:
you mean ensure the independence between the two adapters without affecting each other.Is it?
If so,I think it is good,I will modify